src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color $transition-duration;
+
+    &:hover {
+      color: #00acdc;
```

Comment:
you can add a variable for the font as well. Yeah, you use it only once here, but it's good thing to practice creating it. Most likely that you will do it when you work on the real project